src/Widgets/WorkspaceClone.vala

```diff
@@ -266,7 +266,7 @@ namespace Gala {
                 remove_window (window);
         }
 
-        void update_size (Meta.Rectangle monitor_geometry) {
+        public void update_size (Meta.Rectangle monitor_geometry) {
             if (window_container.width != monitor_geometry.width || window_container.height != monitor_geometry.height) {
                 window_container.set_size (monitor_geometry.width, monitor_geometry.height);
                 background.set_size (window_container.width, window_container.height);
```

Comment:
It might be a good idea to expose the `animate` parameter in `BackgroundManager.set_size` and default it to `true`.

Right now we only call it here (and we happen to want to avoid the animation) but in the future this might change.

Reply:
I did actually have this parameter there for a moment as well but ultimately decided against this, thinking that my expectation of an "update_size" function would be that things happen right away. Of course, having the `animate` parameter there, defaulting to false, won't hurt and makes this explicit. I'd say we see when the use case comes up and only then extend the API :)